app/views/groups/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<main class="body-container">
+  <section class="container">
+   <p><strong>Total transactions:</strong> $<%= @group.entities.sum(:amount) %></p> 
+    <div class="group-info">
+      <% if @group.icon.attached? %>
+        <%= image_tag @group.icon, class: "group-icon" %>
+      <% else %>
+        <%= image_tag "default_product_image.webp", class: "group-icon" %>
+      <% end %>
+    </div>
+
+    <p><strong>Created at:</strong> <%= @group.created_at.strftime('%B %d, %Y at %I:%M %p') %> UTC</p>
+
+
+    <p><strong>Transactions:</strong></p>
+
+    <%= link_to "See all transactions", user_group_entities_path(current_user, @group), class: "all-transactions" %>
```

Comment:
Great design @ONJoseph 🤝

- [ ] As requested by the previous reviewers, kindly ensure your design matches the design provided.
- [ ] Let's try to match the colors, font weight, alignment, or positioning. Some elements of your design are overlapped. You may also fix this. The buttons should also look like the ones provided design.
- Your design
![transactions](https://github.com/ONJoseph/BudgetApp/assets/63917725/0bc3ec5e-03d6-41e2-83f8-0cb8edbc4507)
- Given design.
![Given design](https://github.com/ONJoseph/BudgetApp/assets/63917725/7a3fe125-9d2d-4733-8451-ae126db10c1a)

Reply:
@1GORDON , Thanks a lot for the review. I have worked hard to implemented the changes. Please kindly approve to enable me to progress in the program and not repeat the week a second time. Thanks for the support!

Reply:
- Well resolved 👍🏽 
<img width="459" alt="Screenshot 2023-05-13 at 00 21 34" src="https://github.com/ONJoseph/BudgetApp/assets/58692590/7f7b0d1a-8b93-4a21-9547-fc50f9be615f">